review-notifications/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "review-notifications",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "enzyme": "^3.9.0",
```

Comment:
This along with `enzyme-adapter-react-16` and `react-test-renderer` belongs to `devDependencies`. These packages are used only for development purposes and are not required for the project to function properly.